gradoop-temporal/src/test/java/org/gradoop/temporal/model/impl/TemporalGraphTest.java

```diff
@@ -193,4 +198,25 @@ public void testFromLogicalGraph() throws Exception {
     loadedVertices.forEach(this::checkDefaultTemporalElement);
     loadedEdges.forEach(this::checkDefaultTemporalElement);
   }
+
+  /**
+   * Test the
+   * {@link TemporalGraph#fromLogicalGraph(LogicalGraph, TimeIntervalExtractor, TimeIntervalExtractor, TimeIntervalExtractor)}  } method.
+   */
```

Comment:
There is a `@throws` tag missing for this method

Reply:
added